doc/ics/yaml/recology_com.yaml

```diff
@@ -0,0 +1,14 @@
+---
+title: Recology San Francisco
+url: https://www.recology.com/
+country: us
+howto:
+  en: |
+    - Go to <https://www.recology.com/recology-san-francisco/collection-calendar/>
+    - Enter your home address in the search field and click "Search"
+    - Click on "ADD TO MY CALENDAR" and select "iCal"
+    - Right-click on the iCal link and copy the link address
+    - Replace the `url` in the example configuration with this link
```

Comment:
I don't like the ``Replace the `url` in the example configuration with this link`` line anymore, as it is also shown in the configuration GUI.

I know a lot of older ics sources have this line but I try to replace it whenever I touch one.


I'd prefer something like ``use this url as the `url` attribute``